arch/sim/include/limits.h

```diff
@@ -54,7 +54,8 @@
 
 /* These change on 32-bit and 64-bit platforms */
 
-#if !defined(CONFIG_HOST_X86_64) || defined(CONFIG_SIM_M32)
+#if (!defined(CONFIG_HOST_X86_64) || defined(CONFIG_SIM_M32)) && \
+     !defined(CONFIG_HOST_ARM64)
```

Comment:
let's use the positive logic like line 107:
```
#if (defined(CONFIG_HOST_X86_64) && !defined(CONFIG_SIM_M32)) || \
      defined(CONFIG_HOST_ARM64)
```

Reply:
done